app/controllers/organisation_session_controller.rb

```diff
@@ -0,0 +1,15 @@
+class OrganisationSessionController < ApplicationController
+  include Secured
+
+  def update
+    desired_organisation_id = params[:current_user_organisation]
+
+    if desired_organisation_id
+      if current_user.additional_organisations.pluck(:id).include?(desired_organisation_id) ||
```

Comment:
Could this be expressed as?

```
if current_user.all_organisations.pluck(:id).include?(desired_organisation_id)
```

Or extracted to something like `desired_organisation_is_in_additional_organisations?`